codec-classes-quic/src/main/java/io/netty/incubator/codec/quic/QuicSslContextBuilder.java

```diff
@@ -179,6 +180,14 @@ public QuicSslContextBuilder earlyData(boolean enabled) {
         return this;
     }
 
+    /**
+     * Enable / disable client session cache.
+     */
+    public QuicSslContextBuilder clientSessionCache(boolean enabled) {
```

Comment:
why not use `sessionCache(...)` and allow to enable / disable for client and server ?